app/views/content/landing/how-to-fund-your-teacher-training/_content.html.erb

```diff
@@ -0,0 +1,109 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">How can I fund my teacher training?</h2>
+
+    <div class="inset">
+      <p>
+        Different bursaries and scholarships are available depending on the subject you're training to teach.
```

Comment:
Is this line needed @EllieNodder ? It is basically a repeat of the table caption under the postgrad h3?